CHANGELOG.md

```diff
@@ -1,5 +1,97 @@
 # Release Notes
 
+## Version 1.32.0-beta0 - 2023-10-27
+
+Features:
+* Kicbase: New `--gpus=nvidia` flag to enable NVIDIA container runtime (GPU container support) [#15927](https://github.com/kubernetes/minikube/pull/15927) [#17314](https://github.com/kubernetes/minikube/pull/17314) [#17488](https://github.com/kubernetes/minikube/pull/17488)
```

Comment:
lets change "Nvidia GPU support through  new flag `--gpus=nvidia`  for docker driver.

Reply:
I'll change it once tests come back